zinit.zsh

```diff
@@ -1978,28 +1976,27 @@ builtin setopt noaliases
     fi
     local append influx in_prepend
     if [[ $2 == (b|u|it|st|nb|nu|nit|nst) ]]; then
-        # Powtórzenie kodu aby zabezpieczyć ewentualne krańcowe białe znaki
-        # oraz aby umożliwić akumulację tego kodu z innymi.
+        # Code repetition to protect any extreme whitespace
+        # and to allow accumulation of this code with others.
         append=$ZINIT[col-$2]
     elif [[ $2 == (…|ndsh|mdsh|mmdsh|-…|lr|) || -z $2 || -z $ZINIT[col-$2] ]]; then
-        # Wznowienie poprzedniego kodu escape, jeżeli jest taki zapisany.
+        # Resume previous escape code, if one is present.
         if [[ $ZINIT[__last-formatter-code] != (…|ndsh|mdsh|mmdsh|-…|lr|rst|nl|) ]]; then
             in_prepend=$ZINIT[col-$ZINIT[__last-formatter-code]]
             influx=$ZINIT[col-$ZINIT[__last-formatter-code]]
         fi
-        # W przeciwnym razie brak akcji – tylko skopiowanie
-        # tego kodu bez koloru.
+        # Otherwise no action - only copy of this code without color.
     else
-        # Zakończenie aktywności kodu escape.
+        # Termination of escape code activity.
         append=$ZINIT[col-rst]
     fi
 
-    # Skonstruuj tekst.
+    # Construct the text.
     REPLY=$in_prepend${ZINIT[col-$2]:-$1}$influx$3$append
 
-    # Zamień nowe linie na znaki, które działają tak samo ale nie są
-    # usuwane w podstawieniu $( … ) – vertical tab 0xB ↔ 13 w systemie
-    # oktagonalnym połączone z powrotem karetki (015).
+    # Replace new lines with characters that work the same but are not
+    # deleted in the substitution $ (...) - vertical tab 0xB ↔ 13 in the system
+    # octagonal connected back carriage (015).
```

Comment:
Uh...?

Reply:
Yeah, no idea why we're doing this so I literally just copy+pasted the comment from Google Translate.

Reply:
Functionally, it replaces `\013` with `\015`, but WHY? idk

Reply:
hm. Let's deal with that another time :)